ros2service/ros2service/verb/type.py

```diff
@@ -27,6 +28,7 @@ def add_arguments(self, parser, cli_name):
             help="Name of the ROS service to get type (e.g. '/talker/list_parameters')")
         arg.completer = ServiceNameCompleter(
             include_hidden_services_key='include_hidden_services')
+        add_strategy_node_arguments(parser)
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.